e2e/tests/api-driven/src/demo-workspace/steps/navigation_steps.ts

```diff
@@ -0,0 +1,48 @@
+import { When } from "@cucumber/cucumber";
+import { $admin } from "../../client";
+import { strict as assert } from "node:assert";
+import { getFlowBySlug, getTeamAndFlowsBySlug, getTeams } from "../helper";
+import { CustomWorld } from "./background_steps";
+
+When<CustomWorld>("I am in the {string} team", async function (this, string) {
```

Comment:
naming: Wondering if there is a better variable name other than `string` as this could look a little confusing as a name in a typescript repo... maybe just `teamName`? Or `teamSlug`